tools/appveyor.psm1

```diff
@@ -474,6 +475,19 @@ function Invoke-AppveyorFinish
             $preReleaseVersion = "$previewPrefix-$previewLabel.$env:APPVEYOR_BUILD_NUMBER"
         }
 
+        # Smoke Test MSI installer
+        Write-Verbose "Smoke-Testing MSI installer" -Verbose
+        $msi = $artifacts | Where-Object { $_.EndsWith(".msi") }
+        $msiLog = Join-Path (Get-Location) 'msilog.txt'
+        $msiExecProcess = Start-Process msiexec.exe -Wait -ArgumentList "/I $msi /quiet /l*vx $msiLog" -NoNewWindow -PassThru
```

Comment:
Does it run with elevated privileges?

Reply:
Must do because otherwise it would not work

Reply:
Yes (otherwise it would fail)